src/errors_handler.py

```diff
@@ -0,0 +1,46 @@
+from collections import Counter
+from src.translation import Translation
+
+
+def count_inflectional(translation):
+    number_of_errors = 0
+    translation.remove_correct_translation()
+    lemmas = translation.count_all_lemmas()
+    for word, count in list(lemmas.items()):
+        if count == 1:
```

Comment:
why do we need it? if the count is 1 it is 1 in the ref and 0 in the hyp (or the opposite) and then the min is 0